frontend/testing/cypress/src/selectors/gitea.js

```diff
@@ -4,7 +4,7 @@ export const gitea = {
   getAltinnLogo: () => cy.findByAltText(/altinn(|-| )logo/i),
   getDeleteButton: () => cy.findByRole('button', { name: /delete this repository/i }),
   getDeleteRepositoryModal,
-  getDeleteRepositoryNameField: () => cy.get('#delete-repo-modal #repo_name'),
+  getDeleteRepositoryNameField: () => cy.get('#delete-repo-modal #repo_name_to_delete'),
```

Comment:
Since the ID issue within Gitea is fixed now, we can improve this selector as following:
```suggestion
  getDeleteRepositoryNameField: () => cy.findByLabel(/Navn på repository/i)
```
You can decide if you want to improve it or not. This will be replaced by Playwright in the future.

Reply:
I'll leave it for now as it is.